keep-ui/app/nav.tsx

```diff
@@ -1,7 +1,8 @@
 import Navbar from './navbar';
 import { getServerSession } from '../utils/customAuth';
+import { authOptions } from '../pages/api/auth/[...nextauth]';
```

Comment:
authOptions  should move to authUtils? (and then pages/api/auth will consume too)

Reply:
What is authUtils? can't seem to find anything like that. Took this from the official NextAuth documentation
https://next-auth.js.org/configuration/nextjs#getserversession

Reply:
I mean customAuth

Reply:
Nope